coronavirus_dashboard_summary/Views/Views.HomePage.fs

```diff
@@ -20,12 +20,20 @@ let index (date: Release) (redis: Redis.Client) =
         |> List.groupBy Filters.GroupByMetric
         |> List.map Filters.GroupByPriorityAttribute
         |> Metrics.GeneralPayload
+
+    let dbData =
+        if dbResp.Count = 0 then HomePageModel.Data date Metrics.PostCodeMetrics redis
+                                 |> Async.RunSynchronously
+                                 |> List.groupBy Filters.GroupByMetric
+                                 |> List.map Filters.GroupByPriorityAttribute
+                                 |> Metrics.GeneralPayload
+        else dbResp
```

Comment:
I realized this fix can help only if the relevant data is in the DB (maybe the SQL query string could be adjusted). Using the prod DB probably it should work (it might be slow), but on any other DBs it might be different.